LearningMissionSimulation/PlaygroundMher.cs

```diff
@@ -382,6 +376,17 @@ void UpdateStudentList(Classroom classroom)
         }
 
 
+        public void Clear()
```

Comment:
The Clear method should clear all internal data structures:
ActiveStudentList.Clear();
StudentDictionary.Clear();
SubjectIdList.Clear();
 and so on.